src/Sylius/Bundle/ShopBundle/Resources/views/Common/Form/_address.html.twig

```diff
@@ -1,13 +1,15 @@
-{% if type is not defined %}
-     {% set type = null %}
+{% if type is defined %}
+    {% set type = type ~ '-' %}
```

Comment:
?
IMO you should revert that

Reply:
It was added by @AdamKasp for tests few days ago, It doesn't create BC-break, maybe we should sign it that it is using only for tests?

Reply:
Probably I don't understand you, but you could revert changes in this file and everything should work well

Reply:
Maybe the creation of a separate twig filter could make the biggest sense?